.theme/layouts/default.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="App">
+    <header>
+      <nav class="App__nav">
```

Comment:
bir de ek is yazayim aklima geldi.  nav'dan once mouseless logosu ile nuxt logosunu yan yana koyalim diyorum. en tepede yan yana, ornek :point_down: 

<img src="https://raw.githubusercontent.com/mouseless/brand/main/assets/logo/svg/logo-mark-primary.svg" height="10" /> | <img src="https://nuxt.com/assets/design-kit/logo/icon-green.svg" height="10" />